_posts/2020-12-19-fixing-apple-smart-keyboard-with-copper-tape.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "Fixing Apple Smart Keyboard with Copper Tape"
+featured: true
+author: sivachandran
+comments: true
+tags: [ electronics ]
+---
+
+This is a story of how I fixed my one-year-old Apple Smart Keyboard with inexpensive Copper tape. There are multiple YouTube videos(linked below) that describe the procedure with the use of soldering wires on the connector pins. But, I was able to fix without any soldering and would like to share the steps in this post. 
+
+## Problem
+
+I bought an Apple iPad Air 3 with Apple Smart Keyboard in November 2019. Exactly after one year of use, around mid of November 2019, the keyboard stopped getting recognized by the iPad randomly. Initially, a little adjustment on the Smart Connector pins made it work. But it stopped working entirely soon after. Internet search told me it is a common problem, and the most common fix is cleaning connector pins with an alcohol-based solution. I tried the same, but it didn't help.
+
+I took the keyboard to Apple authorized service centre, which is run by the same company from where bought the iPad Air and Smart Keyboard originally. They first confirmed the problem is on the keyboard by connecting the keyboard to different iPad and connecting a different keyboard to my iPad.
+
+After confirming the problem is on the keyboard, they checked the warranty and realized it has expired one month back. Apparently, Apple has a replacement program for Smart Keyboard(they didn't say it was a manufacturing defect, but I wonder why else they have replacement program). But, they are not sure whether my keyboard is covered under the program. They asked me to hand over the keyboard and said they would call me once they hear back from Apple. I got a call in two days, and I was told Apple can't provide a replacement. I insisted on providing a replacement or at least repair it as it is just one month past from the warranty period. They said they can neither provide replacement and nor repair as these keyboards non-serviceable. The only option they offered me was buying a new Apple Smart Keyboard, which would cost around Rs.14,000.
```

Comment:
You use past tense in the first line and third line but present tense in the second. Specifically `they are not sure whether my keyboard is..` should be `they were not sure whether my keyboard was...`. 

Also --> `They said they can neither provide replacement and nor repair as these keyboards non-serviceable.` here you can drop the `and`